public/js/components/Editor.js

```diff
@@ -87,6 +87,29 @@ const Editor = React.createClass({
     });
 
     this.editor.on("gutterClick", this.onGutterClick);
+
+    // Disable APZ for source editors. It currently causes the line numbers to
+    // "tear off" and swim around on top of the content. Bug 1160601 tracks
+    // finding a solution that allows APZ to work with CodeMirror.
+    this.editor.getScrollerElement().addEventListener("wheel", ev => {
```

Comment:
I think the onWheel logic should land in `util/editor.js` as it's a utility for the editor exclusively.

`this.editor.getScrollerElement().addEventListener("wheel", ev =>  onWheel(this.editor, ev);`